cores/esp8266/Updater.h

```diff
@@ -141,18 +148,35 @@ class UpdaterClass {
       return written;
     }
 
+#ifdef VERIFY_SIGNATURE
+    int addCA(const uint8_t *cert, int *len);
```

Comment:
Similar to `WiFiClientSecure`, suggest exposing a helper template method which loads CA certificate from a file (Stream).